scripts/create_test

```diff
@@ -159,44 +193,43 @@ OR
     parser.add_argument("-q", "--queue", default=None,
                         help="Force batch system to use a certain queue")
 
-    parser.add_argument("--xml-machine",
-                        help="Use this machine key in the lookup in testlist.xml, default is all if any --xml- argument is used")
-    parser.add_argument("--xml-compiler",
-                        help="Use this compiler key in the lookup in testlist.xml, default is all if any --xml- argument is used")
-    parser.add_argument("--xml-category",
-                        help="Use this category key in the lookup in testlist.xml, default is all if any --xml- argument is used")
-    parser.add_argument("--xml-testlist",
-                        help="Use this testlist to lookup tests, default specified in config_files.xml")
-    parser.add_argument("--testfile",
+    parser.add_argument("-f", "--testfile",
                         help="A file containing an ascii list of tests to run")
 
     parser.add_argument("-o", "--allow-baseline-overwrite", action="store_true",
                         help="If the generate baseline option is specified with a baseline directory option "
-                        "existing tests in that directory will not be overwritten unless this flag is provided.")
+                        "existing tests in that directory will not be overwritten unless this flag is provided."
+                        "You can use bless-test-results to bless baselines from a recent run.")
 
     args = parser.parse_args(args[1:])
 
     CIME.utils.handle_standard_logging_options(args)
 
     # generate and compare flags may not point to the same directory
-    if args.generate is not None:
-        expect(not (args.generate == args.compare),
-               "Cannot generate and compare baselines at the same time")
-    expect(not (args.baseline_name is not None and (not args.compare and not args.generate)),
-           "Provided baseline name but did not specify compare or generate")
+    if model == "cesm":
```

Comment:
CESM/ACME differences are handled here in order to cut down complexity in TestScheduler.